js/build.js

```diff
@@ -257,6 +295,40 @@ Fliplet.Widget.instance('login-ds', function(data) {
     // EVENT LISTENER FOR FORGET PASSWORD RESET
     // Just switches views Login to Email verification
     // Leave as it is
+    $newPasswordInput.on('focus', function() {
+      $newPasswordChecker.removeClass('hidden');
+      calculateElHeight($('.state.present'));
+    }).on('blur', function() {
+      if (!$newPasswordInput.val()) {
+        $newPasswordChecker.addClass('hidden');
+        calculateElHeight($('.state.present'));
+      }
+    });
+
+    $confirmPasswordInput.on('focus', function() {
+      $confirmPasswordChecker.removeClass('hidden');
+      calculateElHeight($('.state.present'));
+    }).on('blur', function() {
+      if (!$confirmPasswordInput.val()) {
+        $confirmPasswordChecker.addClass('hidden');
+        calculateElHeight($('.state.present'));
+      }
+    }).on('input', function() {
+      var password = $newPasswordInput.val();
```

Comment:
The content of this function can be moved to a `validatePasswordConfirmation()` function